### PR TITLE
Requires at init-time, before any other evaluation.

### PR DESCRIPTION
**Pull request by tomasz-biernacki**
As per title

### CODE REVIEW

File: src/cljs/replumb/repl.cljs [hunk 1]
```diff
@@ -937,6 +959,13 @@
 
   * :static-fns - static dispatch in generated JavaScript.
 
+  * :requires - a map containing a set of namespaces to load before any other
+  evaluation and a callback cb to call upon completion. The nss map will
+  contain keys that will match the directive type (:require, :use,
+  :require-macros, ect) and values that will be a set of specs of namespaces
+  to load. The format of the values is analogous to the ClojureScript ns form,
+  e.g. '[my.ns :refer [v1 f1]].
```
Comment (arichiardi) [inline src/cljs/replumb/repl.cljs]: I think we should stick with the familiar `(` syntax here

File: src/cljs/replumb/repl.cljs [hunk 2]
```diff
@@ -354,7 +354,7 @@
   #{:verbose :warning-as-error :target :init-fn!
     :no-pr-str-on-value :load-fn! :read-file-fn!
     :write-file-fn! :src-paths :cache :context
-    :foreign-libs :static-fns})
+    :foreign-libs :static-fns :requires})
```
Comment (arichiardi) [inline src/cljs/replumb/repl.cljs]: given how things are now, maybe `:init` is a better name

File: src/cljs/replumb/repl.cljs (hunk 1 above)
Comment (arichiardi) [inline src/cljs/replumb/repl.cljs]: My bad ;)

Comment (tomasz-biernacki) [inline src/cljs/replumb/repl.cljs]: 👍

**Pull request merged**